audio-worklet/src/noisegate-script-processor.js

```diff
@@ -80,7 +80,7 @@ class NoiseGate {
     // The last weight (between 0 and 1) assigned, where 1 means the gate
     // is open and 0 means it is closed and the sample in the output buffer is
     // muted. When attacking, the weight will linearly decrease from 1 to 0, and
-    // when releasing the weight linearly increase from 0 to 1. 
+    // when releasing the weight linearly increase from 0 to 1.
```

Comment:
Hmm.  When attacking shouldn't the weight go from 0 to 1 and 1 to 0 for release?